buildSrc/src/main/java/com/osacky/flank/gradle/YamlWriter.kt

```diff
@@ -24,18 +24,18 @@ internal class YamlWriter {
   }
 
   internal fun writeFlankProperties(config: FladleConfig): String = buildString {
-    val testShards = config.testShards
-    val shardTime = config.shardTime
-    val repeatTests = config.repeatTests
-    val smartFlankGcsPath = config.smartFlankGcsPath
-    val filesToDownload = config.filesToDownload
-    val projectId = config.projectId
+    val testShards = config.testShards.orNull
```

Comment:
i think it might be more idiomatic to check `testShards.isPresent` on line 42.
i have a preference against using `null` to represent a state or value. :)